packages/core/src/odata/v4/filter-functions.ts

```diff
@@ -36,4 +38,31 @@ export function matchesPattern<EntityT extends Entity>(
   return filterFunction('matchesPattern', 'bool', str, regex);
 }
 
+/* Date Functions */
+/**
+ * Build a filter function to get the fractional seconds of a date. Evaluates to int.
+ * @param date - The date to get the fractional seconds for. This can either be a date (Moment) or a reference to a field.
+ * @returns The newly created filter function
+ */
+export function fractionalSeconds<EntityT extends Entity>(
+  date: Moment | Field<EntityT>
+): NumberFilterFunction<EntityT> {
+  return filterFunction('fractionalseconds', 'decimal', date);
```

Comment:
Just to be on the safe side that I understood it correctly. The function inself like `fractionalseconds` is evaluated on the server side on the date field. So you only pass `frantionalsecondes(someDateFiled) eq 1234` in the URL, so there is no conversion needed right?

Reply:
Mostly correct, but there is conversion needed (this is also why you have to pass 'decimal'). What we pass would rather be this: `fractionalseconds(someDateFiled) eq 1234D` <= the value you compare with has to be converted.